CRITICAL - 2024-11-21 03:43:31 --> ErrorException: Maximum execution time of 120 seconds exceeded
[Method: POST, Route: login]
in SYSTEMPATH\Debug\Exceptions.php on line 209.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
ERROR - 2024-11-21 03:47:15 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'login' in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'login', 3306, '', 0)
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp82\htdocs\codeigniter4\app\Models\Usuarios.php(8): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp82\htdocs\codeigniter4\app\Controllers\Home.php(21): App\Models\Usuarios->obtenerUsuario(Array)
#6 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Home->login()
#7 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp82\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'login' in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp82\htdocs\codeigniter4\app\Models\Usuarios.php(8): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp82\htdocs\codeigniter4\app\Controllers\Home.php(21): App\Models\Usuarios->obtenerUsuario(Array)
#5 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Home->login()
#6 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp82\htdocs\codeigniter4\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp82\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-11-21 03:47:15 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'login'
[Method: POST, Route: login]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `t_usuario`
WHERE `usuario` = :usuario:', [...], false)
 3 APPPATH\Models\Usuarios.php(8): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Home.php(21): App\Models\Usuarios->obtenerUsuario([...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->login()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-21 03:47:18 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'login' in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'login', 3306, '', 0)
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp82\htdocs\codeigniter4\app\Models\Usuarios.php(8): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp82\htdocs\codeigniter4\app\Controllers\Home.php(21): App\Models\Usuarios->obtenerUsuario(Array)
#6 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Home->login()
#7 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp82\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'login' in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp82\htdocs\codeigniter4\app\Models\Usuarios.php(8): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp82\htdocs\codeigniter4\app\Controllers\Home.php(21): App\Models\Usuarios->obtenerUsuario(Array)
#5 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Home->login()
#6 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp82\htdocs\codeigniter4\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp82\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-11-21 03:47:18 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'login'
[Method: POST, Route: login]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `t_usuario`
WHERE `usuario` = :usuario:', [...], false)
 3 APPPATH\Models\Usuarios.php(8): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Home.php(21): App\Models\Usuarios->obtenerUsuario([...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->login()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-21 03:47:29 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'login' in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'login', 3306, '', 0)
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp82\htdocs\codeigniter4\app\Models\Usuarios.php(8): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp82\htdocs\codeigniter4\app\Controllers\Home.php(21): App\Models\Usuarios->obtenerUsuario(Array)
#6 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Home->login()
#7 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp82\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'login' in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp82\htdocs\codeigniter4\app\Models\Usuarios.php(8): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp82\htdocs\codeigniter4\app\Controllers\Home.php(21): App\Models\Usuarios->obtenerUsuario(Array)
#5 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Home->login()
#6 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp82\htdocs\codeigniter4\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp82\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-11-21 03:47:29 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'login'
[Method: POST, Route: login]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `t_usuario`
WHERE `usuario` = :usuario:', [...], false)
 3 APPPATH\Models\Usuarios.php(8): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Home.php(21): App\Models\Usuarios->obtenerUsuario([...])
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->login()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-21 03:52:58 --> mysqli_sql_exception: Table '`login`.`migrations`' already exists in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(806): CodeIgniter\Database\Forge->createTable()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(163): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest()
#7 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#11 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#12 {main}
ERROR - 2024-11-21 03:52:58 --> mysqli_sql_exception: Table 'login.migrations' doesn't exist in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(653): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(176): CodeIgniter\Database\MigrationRunner->getHistory()
#6 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest()
#7 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#11 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#12 {main}
ERROR - 2024-11-21 03:52:58 --> mysqli_sql_exception: Table 'login.migrations' doesn't exist in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(701): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(180): CodeIgniter\Database\MigrationRunner->getLastBatch()
#6 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest()
#7 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#11 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#12 {main}
ERROR - 2024-11-21 03:54:37 --> mysqli_sql_exception: Table 'login.t_usuario' doesn't exist in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp82\htdocs\codeigniter4\app\Models\Usuarios.php(8): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp82\htdocs\codeigniter4\app\Controllers\Home.php(21): App\Models\Usuarios->obtenerUsuario(Array)
#6 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Home->login()
#7 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp82\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
ERROR - 2024-11-21 03:54:37 --> La consulta a la base de datos falló en obtenerUsuario.
CRITICAL - 2024-11-21 03:54:37 --> TypeError: count(): Argument #1 ($value) must be of type Countable|array, bool given
[Method: POST, Route: login]
in APPPATH\Controllers\Home.php on line 23.
 1 APPPATH\Controllers\Home.php(23): count(false)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->login()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-21 03:59:59 --> mysqli_sql_exception: Tablespace for table '`login`.`migrations`' exists. Please DISCARD the tablespace before IMPORT in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(806): CodeIgniter\Database\Forge->createTable()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(163): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest()
#7 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#11 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#12 {main}
ERROR - 2024-11-21 03:59:59 --> mysqli_sql_exception: Table 'login.migrations' doesn't exist in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(653): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(176): CodeIgniter\Database\MigrationRunner->getHistory()
#6 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest()
#7 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#11 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#12 {main}
ERROR - 2024-11-21 03:59:59 --> mysqli_sql_exception: Table 'login.migrations' doesn't exist in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(701): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(180): CodeIgniter\Database\MigrationRunner->getLastBatch()
#6 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest()
#7 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#11 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#12 {main}
ERROR - 2024-11-21 04:10:34 --> mysqli_sql_exception: Got error 194 "Tablespace is missing for a table" from storage engine InnoDB in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(653): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(176): CodeIgniter\Database\MigrationRunner->getHistory()
#6 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest()
#7 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#11 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#12 {main}
ERROR - 2024-11-21 04:10:34 --> mysqli_sql_exception: Got error 194 "Tablespace is missing for a table" from storage engine InnoDB in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(701): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(180): CodeIgniter\Database\MigrationRunner->getLastBatch()
#6 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest()
#7 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#11 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#12 {main}
ERROR - 2024-11-21 04:10:42 --> mysqli_sql_exception: Got error 194 "Tablespace is missing for a table" from storage engine InnoDB in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(653): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(176): CodeIgniter\Database\MigrationRunner->getHistory()
#6 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest()
#7 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#11 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#12 {main}
ERROR - 2024-11-21 04:10:42 --> mysqli_sql_exception: Got error 194 "Tablespace is missing for a table" from storage engine InnoDB in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(701): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(180): CodeIgniter\Database\MigrationRunner->getLastBatch()
#6 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest()
#7 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#11 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#12 {main}
ERROR - 2024-11-21 04:12:30 --> mysqli_sql_exception: Got error 194 "Tablespace is missing for a table" from storage engine InnoDB in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(653): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(176): CodeIgniter\Database\MigrationRunner->getHistory()
#6 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest()
#7 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#11 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#12 {main}
ERROR - 2024-11-21 04:12:30 --> mysqli_sql_exception: Got error 194 "Tablespace is missing for a table" from storage engine InnoDB in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(701): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(180): CodeIgniter\Database\MigrationRunner->getLastBatch()
#6 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest()
#7 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#11 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#12 {main}
ERROR - 2024-11-21 04:12:34 --> mysqli_sql_exception: Got error 194 "Tablespace is missing for a table" from storage engine InnoDB in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(653): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(176): CodeIgniter\Database\MigrationRunner->getHistory()
#6 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest()
#7 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#11 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#12 {main}
ERROR - 2024-11-21 04:12:34 --> mysqli_sql_exception: Got error 194 "Tablespace is missing for a table" from storage engine InnoDB in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(701): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(180): CodeIgniter\Database\MigrationRunner->getLastBatch()
#6 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest()
#7 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#11 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#12 {main}
ERROR - 2024-11-21 04:36:04 --> mysqli_sql_exception: Unknown table 'login.t_usuario_roles' in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\Forge.php(663): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\app\Database\Migrations\2024-11-21-041841_TUsuarioRoles.php(38): CodeIgniter\Database\Forge->dropTable()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(867): App\Database\Migrations\TUsuarioRoles->down()
#6 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(290): CodeIgniter\Database\MigrationRunner->migrate()
#7 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\MigrateRollback.php(101): CodeIgniter\Database\MigrationRunner->regress()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\MigrateRollback->run()
#9 C:\xampp82\htdocs\codeigniter4\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\MigrateRefresh.php(86): CodeIgniter\CLI\BaseCommand->call()
#11 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\MigrateRefresh->run()
#12 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#13 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#14 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#15 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#16 {main}
ERROR - 2024-11-21 04:36:04 --> mysqli_sql_exception: Unknown table 'login.rol_permisos' in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\Forge.php(663): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\app\Database\Migrations\2024-11-21-041819_RolPermisos.php(38): CodeIgniter\Database\Forge->dropTable()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(867): App\Database\Migrations\RolPermisos->down()
#6 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(290): CodeIgniter\Database\MigrationRunner->migrate()
#7 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\MigrateRollback.php(101): CodeIgniter\Database\MigrationRunner->regress()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\MigrateRollback->run()
#9 C:\xampp82\htdocs\codeigniter4\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\MigrateRefresh.php(86): CodeIgniter\CLI\BaseCommand->call()
#11 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\MigrateRefresh->run()
#12 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#13 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#14 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#15 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#16 {main}
ERROR - 2024-11-21 04:36:04 --> mysqli_sql_exception: Unknown table 'login.t_usuarios' in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\Forge.php(663): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\app\Database\Migrations\2020-12-04-202039_t_usuarios.php(35): CodeIgniter\Database\Forge->dropTable()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(867): App\Database\Migrations\TUsuarios->down()
#6 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(290): CodeIgniter\Database\MigrationRunner->migrate()
#7 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\MigrateRollback.php(101): CodeIgniter\Database\MigrationRunner->regress()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\MigrateRollback->run()
#9 C:\xampp82\htdocs\codeigniter4\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\MigrateRefresh.php(86): CodeIgniter\CLI\BaseCommand->call()
#11 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\MigrateRefresh->run()
#12 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#13 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#14 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#15 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#16 {main}
ERROR - 2024-11-21 05:05:29 --> ParseError: syntax error, unexpected single-quoted string "SessionUser", expecting "]" in C:\xampp82\htdocs\codeigniter4\app\Config\Filters.php:14
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile()
#1 C:\xampp82\htdocs\codeigniter4\system\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace()
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass()
#3 C:\xampp82\htdocs\codeigniter4\system\Config\Factories.php(297): class_exists()
#4 C:\xampp82\htdocs\codeigniter4\system\Config\Factories.php(216): CodeIgniter\Config\Factories::locateClass()
#5 C:\xampp82\htdocs\codeigniter4\system\Config\Factories.php(159): CodeIgniter\Config\Factories::getDefinedInstance()
#6 C:\xampp82\htdocs\codeigniter4\system\Config\Factories.php(188): CodeIgniter\Config\Factories::__callStatic()
#7 C:\xampp82\htdocs\codeigniter4\system\Common.php(208): CodeIgniter\Config\Factories::get()
#8 C:\xampp82\htdocs\codeigniter4\system\Config\Services.php(295): config()
#9 C:\xampp82\htdocs\codeigniter4\system\Config\BaseService.php(312): CodeIgniter\Config\Services::filters()
#10 C:\xampp82\htdocs\codeigniter4\system\Config\BaseService.php(251): CodeIgniter\Config\BaseService::__callStatic()
#11 C:\xampp82\htdocs\codeigniter4\system\Config\Services.php(292): CodeIgniter\Config\BaseService::getSharedInstance()
#12 C:\xampp82\htdocs\codeigniter4\system\Config\BaseService.php(321): CodeIgniter\Config\Services::filters()
#13 C:\xampp82\htdocs\codeigniter4\system\Config\BaseService.php(202): CodeIgniter\Config\BaseService::__callStatic()
#14 C:\xampp82\htdocs\codeigniter4\system\Common.php(998): CodeIgniter\Config\BaseService::get()
#15 C:\xampp82\htdocs\codeigniter4\system\View\View.php(264): service()
#16 C:\xampp82\htdocs\codeigniter4\system\Common.php(1171): CodeIgniter\View\View->render()
#17 C:\xampp82\htdocs\codeigniter4\system\CLI\GeneratorTrait.php(347): view()
#18 C:\xampp82\htdocs\codeigniter4\system\CLI\GeneratorTrait.php(390): CodeIgniter\Commands\Generators\FilterGenerator->renderTemplate()
#19 C:\xampp82\htdocs\codeigniter4\system\CLI\GeneratorTrait.php(253): CodeIgniter\Commands\Generators\FilterGenerator->parseTemplate()
#20 C:\xampp82\htdocs\codeigniter4\system\CLI\GeneratorTrait.php(400): CodeIgniter\Commands\Generators\FilterGenerator->prepare()
#21 C:\xampp82\htdocs\codeigniter4\system\CLI\GeneratorTrait.php(135): CodeIgniter\Commands\Generators\FilterGenerator->buildContent()
#22 C:\xampp82\htdocs\codeigniter4\system\Commands\Generators\FilterGenerator.php(84): CodeIgniter\Commands\Generators\FilterGenerator->generateClass()
#23 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Generators\FilterGenerator->run()
#24 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#25 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#26 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#27 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#28 {main}
ERROR - 2024-11-21 05:06:53 --> mysqli_sql_exception: Table 'login.t_usuarios' doesn't exist in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1836): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(2225): CodeIgniter\Database\BaseBuilder->batchExecute()
#5 C:\xampp82\htdocs\codeigniter4\app\Database\Seeds\Usuario.php(22): CodeIgniter\Database\BaseBuilder->insertBatch()
#6 C:\xampp82\htdocs\codeigniter4\system\Database\Seeder.php(147): App\Database\Seeds\Usuario->run()
#7 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Seed->run()
#9 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#12 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#13 {main}
CRITICAL - 2024-11-21 05:07:00 --> ParseError: syntax error, unexpected single-quoted string "SessionUser", expecting "]"
[Method: GET, Route: salir]
in APPPATH\Config\Filters.php on line 14.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp82\\htdocs\\codeigniter4\\app\\Config\\Filters.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Filters')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Filters')
 4 SYSTEMPATH\Config\Factories.php(297): class_exists('Config\\Filters')
 5 SYSTEMPATH\Config\Factories.php(216): CodeIgniter\Config\Factories::locateClass([...], 'Config\\Filters')
 6 SYSTEMPATH\Config\Factories.php(159): CodeIgniter\Config\Factories::getDefinedInstance([...], 'Config\\Filters', [])
 7 SYSTEMPATH\Config\Factories.php(188): CodeIgniter\Config\Factories::__callStatic('config', [])
 8 SYSTEMPATH\Common.php(208): CodeIgniter\Config\Factories::get('config', 'Config\\Filters')
 9 SYSTEMPATH\Config\Services.php(295): config('Config\\Filters')
10 SYSTEMPATH\Config\BaseService.php(312): CodeIgniter\Config\Services::filters(null, false)
11 SYSTEMPATH\Config\BaseService.php(251): CodeIgniter\Config\BaseService::__callStatic('filters', [...])
12 SYSTEMPATH\Config\Services.php(292): CodeIgniter\Config\BaseService::getSharedInstance('filters', null)
13 SYSTEMPATH\Config\BaseService.php(321): CodeIgniter\Config\Services::filters()
14 SYSTEMPATH\CodeIgniter.php(346): CodeIgniter\Config\BaseService::__callStatic('filters', [])
15 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
16 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
17 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-21 05:07:06 --> ParseError: syntax error, unexpected single-quoted string "SessionUser", expecting "]"
[Method: GET, Route: inicio]
in APPPATH\Config\Filters.php on line 14.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp82\\htdocs\\codeigniter4\\app\\Config\\Filters.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Filters')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Filters')
 4 SYSTEMPATH\Config\Factories.php(297): class_exists('Config\\Filters')
 5 SYSTEMPATH\Config\Factories.php(216): CodeIgniter\Config\Factories::locateClass([...], 'Config\\Filters')
 6 SYSTEMPATH\Config\Factories.php(159): CodeIgniter\Config\Factories::getDefinedInstance([...], 'Config\\Filters', [])
 7 SYSTEMPATH\Config\Factories.php(188): CodeIgniter\Config\Factories::__callStatic('config', [])
 8 SYSTEMPATH\Common.php(208): CodeIgniter\Config\Factories::get('config', 'Config\\Filters')
 9 SYSTEMPATH\Config\Services.php(295): config('Config\\Filters')
10 SYSTEMPATH\Config\BaseService.php(312): CodeIgniter\Config\Services::filters(null, false)
11 SYSTEMPATH\Config\BaseService.php(251): CodeIgniter\Config\BaseService::__callStatic('filters', [...])
12 SYSTEMPATH\Config\Services.php(292): CodeIgniter\Config\BaseService::getSharedInstance('filters', null)
13 SYSTEMPATH\Config\BaseService.php(321): CodeIgniter\Config\Services::filters()
14 SYSTEMPATH\CodeIgniter.php(346): CodeIgniter\Config\BaseService::__callStatic('filters', [])
15 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
16 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
17 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-21 05:07:07 --> ParseError: syntax error, unexpected single-quoted string "SessionUser", expecting "]"
[Method: GET, Route: /]
in APPPATH\Config\Filters.php on line 14.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp82\\htdocs\\codeigniter4\\app\\Config\\Filters.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Filters')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Filters')
 4 SYSTEMPATH\Config\Factories.php(297): class_exists('Config\\Filters')
 5 SYSTEMPATH\Config\Factories.php(216): CodeIgniter\Config\Factories::locateClass([...], 'Config\\Filters')
 6 SYSTEMPATH\Config\Factories.php(159): CodeIgniter\Config\Factories::getDefinedInstance([...], 'Config\\Filters', [])
 7 SYSTEMPATH\Config\Factories.php(188): CodeIgniter\Config\Factories::__callStatic('config', [])
 8 SYSTEMPATH\Common.php(208): CodeIgniter\Config\Factories::get('config', 'Config\\Filters')
 9 SYSTEMPATH\Config\Services.php(295): config('Config\\Filters')
10 SYSTEMPATH\Config\BaseService.php(312): CodeIgniter\Config\Services::filters(null, false)
11 SYSTEMPATH\Config\BaseService.php(251): CodeIgniter\Config\BaseService::__callStatic('filters', [...])
12 SYSTEMPATH\Config\Services.php(292): CodeIgniter\Config\BaseService::getSharedInstance('filters', null)
13 SYSTEMPATH\Config\BaseService.php(321): CodeIgniter\Config\Services::filters()
14 SYSTEMPATH\CodeIgniter.php(346): CodeIgniter\Config\BaseService::__callStatic('filters', [])
15 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
16 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
17 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-21 05:07:08 --> ParseError: syntax error, unexpected single-quoted string "SessionUser", expecting "]"
[Method: GET, Route: /]
in APPPATH\Config\Filters.php on line 14.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp82\\htdocs\\codeigniter4\\app\\Config\\Filters.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Filters')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Filters')
 4 SYSTEMPATH\Config\Factories.php(297): class_exists('Config\\Filters')
 5 SYSTEMPATH\Config\Factories.php(216): CodeIgniter\Config\Factories::locateClass([...], 'Config\\Filters')
 6 SYSTEMPATH\Config\Factories.php(159): CodeIgniter\Config\Factories::getDefinedInstance([...], 'Config\\Filters', [])
 7 SYSTEMPATH\Config\Factories.php(188): CodeIgniter\Config\Factories::__callStatic('config', [])
 8 SYSTEMPATH\Common.php(208): CodeIgniter\Config\Factories::get('config', 'Config\\Filters')
 9 SYSTEMPATH\Config\Services.php(295): config('Config\\Filters')
10 SYSTEMPATH\Config\BaseService.php(312): CodeIgniter\Config\Services::filters(null, false)
11 SYSTEMPATH\Config\BaseService.php(251): CodeIgniter\Config\BaseService::__callStatic('filters', [...])
12 SYSTEMPATH\Config\Services.php(292): CodeIgniter\Config\BaseService::getSharedInstance('filters', null)
13 SYSTEMPATH\Config\BaseService.php(321): CodeIgniter\Config\Services::filters()
14 SYSTEMPATH\CodeIgniter.php(346): CodeIgniter\Config\BaseService::__callStatic('filters', [])
15 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
16 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
17 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-21 05:07:09 --> ParseError: syntax error, unexpected single-quoted string "SessionUser", expecting "]"
[Method: GET, Route: /]
in APPPATH\Config\Filters.php on line 14.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp82\\htdocs\\codeigniter4\\app\\Config\\Filters.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Filters')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Filters')
 4 SYSTEMPATH\Config\Factories.php(297): class_exists('Config\\Filters')
 5 SYSTEMPATH\Config\Factories.php(216): CodeIgniter\Config\Factories::locateClass([...], 'Config\\Filters')
 6 SYSTEMPATH\Config\Factories.php(159): CodeIgniter\Config\Factories::getDefinedInstance([...], 'Config\\Filters', [])
 7 SYSTEMPATH\Config\Factories.php(188): CodeIgniter\Config\Factories::__callStatic('config', [])
 8 SYSTEMPATH\Common.php(208): CodeIgniter\Config\Factories::get('config', 'Config\\Filters')
 9 SYSTEMPATH\Config\Services.php(295): config('Config\\Filters')
10 SYSTEMPATH\Config\BaseService.php(312): CodeIgniter\Config\Services::filters(null, false)
11 SYSTEMPATH\Config\BaseService.php(251): CodeIgniter\Config\BaseService::__callStatic('filters', [...])
12 SYSTEMPATH\Config\Services.php(292): CodeIgniter\Config\BaseService::getSharedInstance('filters', null)
13 SYSTEMPATH\Config\BaseService.php(321): CodeIgniter\Config\Services::filters()
14 SYSTEMPATH\CodeIgniter.php(346): CodeIgniter\Config\BaseService::__callStatic('filters', [])
15 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
16 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
17 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-21 05:07:09 --> ParseError: syntax error, unexpected single-quoted string "SessionUser", expecting "]"
[Method: GET, Route: /]
in APPPATH\Config\Filters.php on line 14.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp82\\htdocs\\codeigniter4\\app\\Config\\Filters.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Filters')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Filters')
 4 SYSTEMPATH\Config\Factories.php(297): class_exists('Config\\Filters')
 5 SYSTEMPATH\Config\Factories.php(216): CodeIgniter\Config\Factories::locateClass([...], 'Config\\Filters')
 6 SYSTEMPATH\Config\Factories.php(159): CodeIgniter\Config\Factories::getDefinedInstance([...], 'Config\\Filters', [])
 7 SYSTEMPATH\Config\Factories.php(188): CodeIgniter\Config\Factories::__callStatic('config', [])
 8 SYSTEMPATH\Common.php(208): CodeIgniter\Config\Factories::get('config', 'Config\\Filters')
 9 SYSTEMPATH\Config\Services.php(295): config('Config\\Filters')
10 SYSTEMPATH\Config\BaseService.php(312): CodeIgniter\Config\Services::filters(null, false)
11 SYSTEMPATH\Config\BaseService.php(251): CodeIgniter\Config\BaseService::__callStatic('filters', [...])
12 SYSTEMPATH\Config\Services.php(292): CodeIgniter\Config\BaseService::getSharedInstance('filters', null)
13 SYSTEMPATH\Config\BaseService.php(321): CodeIgniter\Config\Services::filters()
14 SYSTEMPATH\CodeIgniter.php(346): CodeIgniter\Config\BaseService::__callStatic('filters', [])
15 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
16 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
17 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-21 05:07:09 --> ParseError: syntax error, unexpected single-quoted string "SessionUser", expecting "]"
[Method: GET, Route: /]
in APPPATH\Config\Filters.php on line 14.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp82\\htdocs\\codeigniter4\\app\\Config\\Filters.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Filters')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Filters')
 4 SYSTEMPATH\Config\Factories.php(297): class_exists('Config\\Filters')
 5 SYSTEMPATH\Config\Factories.php(216): CodeIgniter\Config\Factories::locateClass([...], 'Config\\Filters')
 6 SYSTEMPATH\Config\Factories.php(159): CodeIgniter\Config\Factories::getDefinedInstance([...], 'Config\\Filters', [])
 7 SYSTEMPATH\Config\Factories.php(188): CodeIgniter\Config\Factories::__callStatic('config', [])
 8 SYSTEMPATH\Common.php(208): CodeIgniter\Config\Factories::get('config', 'Config\\Filters')
 9 SYSTEMPATH\Config\Services.php(295): config('Config\\Filters')
10 SYSTEMPATH\Config\BaseService.php(312): CodeIgniter\Config\Services::filters(null, false)
11 SYSTEMPATH\Config\BaseService.php(251): CodeIgniter\Config\BaseService::__callStatic('filters', [...])
12 SYSTEMPATH\Config\Services.php(292): CodeIgniter\Config\BaseService::getSharedInstance('filters', null)
13 SYSTEMPATH\Config\BaseService.php(321): CodeIgniter\Config\Services::filters()
14 SYSTEMPATH\CodeIgniter.php(346): CodeIgniter\Config\BaseService::__callStatic('filters', [])
15 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
16 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
17 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-21 05:07:09 --> ParseError: syntax error, unexpected single-quoted string "SessionUser", expecting "]"
[Method: GET, Route: /]
in APPPATH\Config\Filters.php on line 14.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp82\\htdocs\\codeigniter4\\app\\Config\\Filters.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Filters')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Filters')
 4 SYSTEMPATH\Config\Factories.php(297): class_exists('Config\\Filters')
 5 SYSTEMPATH\Config\Factories.php(216): CodeIgniter\Config\Factories::locateClass([...], 'Config\\Filters')
 6 SYSTEMPATH\Config\Factories.php(159): CodeIgniter\Config\Factories::getDefinedInstance([...], 'Config\\Filters', [])
 7 SYSTEMPATH\Config\Factories.php(188): CodeIgniter\Config\Factories::__callStatic('config', [])
 8 SYSTEMPATH\Common.php(208): CodeIgniter\Config\Factories::get('config', 'Config\\Filters')
 9 SYSTEMPATH\Config\Services.php(295): config('Config\\Filters')
10 SYSTEMPATH\Config\BaseService.php(312): CodeIgniter\Config\Services::filters(null, false)
11 SYSTEMPATH\Config\BaseService.php(251): CodeIgniter\Config\BaseService::__callStatic('filters', [...])
12 SYSTEMPATH\Config\Services.php(292): CodeIgniter\Config\BaseService::getSharedInstance('filters', null)
13 SYSTEMPATH\Config\BaseService.php(321): CodeIgniter\Config\Services::filters()
14 SYSTEMPATH\CodeIgniter.php(346): CodeIgniter\Config\BaseService::__callStatic('filters', [])
15 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
16 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
17 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-21 05:07:51 --> ParseError: syntax error, unexpected single-quoted string "SessionUser", expecting "]"
[Method: GET, Route: /]
in APPPATH\Config\Filters.php on line 14.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp82\\htdocs\\codeigniter4\\app\\Config\\Filters.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Filters')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Filters')
 4 SYSTEMPATH\Config\Factories.php(297): class_exists('Config\\Filters')
 5 SYSTEMPATH\Config\Factories.php(216): CodeIgniter\Config\Factories::locateClass([...], 'Config\\Filters')
 6 SYSTEMPATH\Config\Factories.php(159): CodeIgniter\Config\Factories::getDefinedInstance([...], 'Config\\Filters', [])
 7 SYSTEMPATH\Config\Factories.php(188): CodeIgniter\Config\Factories::__callStatic('config', [])
 8 SYSTEMPATH\Common.php(208): CodeIgniter\Config\Factories::get('config', 'Config\\Filters')
 9 SYSTEMPATH\Config\Services.php(295): config('Config\\Filters')
10 SYSTEMPATH\Config\BaseService.php(312): CodeIgniter\Config\Services::filters(null, false)
11 SYSTEMPATH\Config\BaseService.php(251): CodeIgniter\Config\BaseService::__callStatic('filters', [...])
12 SYSTEMPATH\Config\Services.php(292): CodeIgniter\Config\BaseService::getSharedInstance('filters', null)
13 SYSTEMPATH\Config\BaseService.php(321): CodeIgniter\Config\Services::filters()
14 SYSTEMPATH\CodeIgniter.php(346): CodeIgniter\Config\BaseService::__callStatic('filters', [])
15 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
16 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
17 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-21 05:07:52 --> ParseError: syntax error, unexpected single-quoted string "SessionUser", expecting "]"
[Method: GET, Route: /]
in APPPATH\Config\Filters.php on line 14.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp82\\htdocs\\codeigniter4\\app\\Config\\Filters.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Filters')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Filters')
 4 SYSTEMPATH\Config\Factories.php(297): class_exists('Config\\Filters')
 5 SYSTEMPATH\Config\Factories.php(216): CodeIgniter\Config\Factories::locateClass([...], 'Config\\Filters')
 6 SYSTEMPATH\Config\Factories.php(159): CodeIgniter\Config\Factories::getDefinedInstance([...], 'Config\\Filters', [])
 7 SYSTEMPATH\Config\Factories.php(188): CodeIgniter\Config\Factories::__callStatic('config', [])
 8 SYSTEMPATH\Common.php(208): CodeIgniter\Config\Factories::get('config', 'Config\\Filters')
 9 SYSTEMPATH\Config\Services.php(295): config('Config\\Filters')
10 SYSTEMPATH\Config\BaseService.php(312): CodeIgniter\Config\Services::filters(null, false)
11 SYSTEMPATH\Config\BaseService.php(251): CodeIgniter\Config\BaseService::__callStatic('filters', [...])
12 SYSTEMPATH\Config\Services.php(292): CodeIgniter\Config\BaseService::getSharedInstance('filters', null)
13 SYSTEMPATH\Config\BaseService.php(321): CodeIgniter\Config\Services::filters()
14 SYSTEMPATH\CodeIgniter.php(346): CodeIgniter\Config\BaseService::__callStatic('filters', [])
15 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
16 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
17 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-21 05:07:52 --> ParseError: syntax error, unexpected single-quoted string "SessionUser", expecting "]"
[Method: GET, Route: /]
in APPPATH\Config\Filters.php on line 14.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp82\\htdocs\\codeigniter4\\app\\Config\\Filters.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Filters')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Filters')
 4 SYSTEMPATH\Config\Factories.php(297): class_exists('Config\\Filters')
 5 SYSTEMPATH\Config\Factories.php(216): CodeIgniter\Config\Factories::locateClass([...], 'Config\\Filters')
 6 SYSTEMPATH\Config\Factories.php(159): CodeIgniter\Config\Factories::getDefinedInstance([...], 'Config\\Filters', [])
 7 SYSTEMPATH\Config\Factories.php(188): CodeIgniter\Config\Factories::__callStatic('config', [])
 8 SYSTEMPATH\Common.php(208): CodeIgniter\Config\Factories::get('config', 'Config\\Filters')
 9 SYSTEMPATH\Config\Services.php(295): config('Config\\Filters')
10 SYSTEMPATH\Config\BaseService.php(312): CodeIgniter\Config\Services::filters(null, false)
11 SYSTEMPATH\Config\BaseService.php(251): CodeIgniter\Config\BaseService::__callStatic('filters', [...])
12 SYSTEMPATH\Config\Services.php(292): CodeIgniter\Config\BaseService::getSharedInstance('filters', null)
13 SYSTEMPATH\Config\BaseService.php(321): CodeIgniter\Config\Services::filters()
14 SYSTEMPATH\CodeIgniter.php(346): CodeIgniter\Config\BaseService::__callStatic('filters', [])
15 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
16 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
17 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-21 05:07:52 --> ParseError: syntax error, unexpected single-quoted string "SessionUser", expecting "]"
[Method: GET, Route: /]
in APPPATH\Config\Filters.php on line 14.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp82\\htdocs\\codeigniter4\\app\\Config\\Filters.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Filters')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Filters')
 4 SYSTEMPATH\Config\Factories.php(297): class_exists('Config\\Filters')
 5 SYSTEMPATH\Config\Factories.php(216): CodeIgniter\Config\Factories::locateClass([...], 'Config\\Filters')
 6 SYSTEMPATH\Config\Factories.php(159): CodeIgniter\Config\Factories::getDefinedInstance([...], 'Config\\Filters', [])
 7 SYSTEMPATH\Config\Factories.php(188): CodeIgniter\Config\Factories::__callStatic('config', [])
 8 SYSTEMPATH\Common.php(208): CodeIgniter\Config\Factories::get('config', 'Config\\Filters')
 9 SYSTEMPATH\Config\Services.php(295): config('Config\\Filters')
10 SYSTEMPATH\Config\BaseService.php(312): CodeIgniter\Config\Services::filters(null, false)
11 SYSTEMPATH\Config\BaseService.php(251): CodeIgniter\Config\BaseService::__callStatic('filters', [...])
12 SYSTEMPATH\Config\Services.php(292): CodeIgniter\Config\BaseService::getSharedInstance('filters', null)
13 SYSTEMPATH\Config\BaseService.php(321): CodeIgniter\Config\Services::filters()
14 SYSTEMPATH\CodeIgniter.php(346): CodeIgniter\Config\BaseService::__callStatic('filters', [])
15 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
16 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
17 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-21 05:07:52 --> ParseError: syntax error, unexpected single-quoted string "SessionUser", expecting "]"
[Method: GET, Route: /]
in APPPATH\Config\Filters.php on line 14.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp82\\htdocs\\codeigniter4\\app\\Config\\Filters.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Filters')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Filters')
 4 SYSTEMPATH\Config\Factories.php(297): class_exists('Config\\Filters')
 5 SYSTEMPATH\Config\Factories.php(216): CodeIgniter\Config\Factories::locateClass([...], 'Config\\Filters')
 6 SYSTEMPATH\Config\Factories.php(159): CodeIgniter\Config\Factories::getDefinedInstance([...], 'Config\\Filters', [])
 7 SYSTEMPATH\Config\Factories.php(188): CodeIgniter\Config\Factories::__callStatic('config', [])
 8 SYSTEMPATH\Common.php(208): CodeIgniter\Config\Factories::get('config', 'Config\\Filters')
 9 SYSTEMPATH\Config\Services.php(295): config('Config\\Filters')
10 SYSTEMPATH\Config\BaseService.php(312): CodeIgniter\Config\Services::filters(null, false)
11 SYSTEMPATH\Config\BaseService.php(251): CodeIgniter\Config\BaseService::__callStatic('filters', [...])
12 SYSTEMPATH\Config\Services.php(292): CodeIgniter\Config\BaseService::getSharedInstance('filters', null)
13 SYSTEMPATH\Config\BaseService.php(321): CodeIgniter\Config\Services::filters()
14 SYSTEMPATH\CodeIgniter.php(346): CodeIgniter\Config\BaseService::__callStatic('filters', [])
15 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
16 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
17 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-21 05:07:53 --> ParseError: syntax error, unexpected single-quoted string "SessionUser", expecting "]"
[Method: GET, Route: /]
in APPPATH\Config\Filters.php on line 14.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp82\\htdocs\\codeigniter4\\app\\Config\\Filters.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Filters')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Filters')
 4 SYSTEMPATH\Config\Factories.php(297): class_exists('Config\\Filters')
 5 SYSTEMPATH\Config\Factories.php(216): CodeIgniter\Config\Factories::locateClass([...], 'Config\\Filters')
 6 SYSTEMPATH\Config\Factories.php(159): CodeIgniter\Config\Factories::getDefinedInstance([...], 'Config\\Filters', [])
 7 SYSTEMPATH\Config\Factories.php(188): CodeIgniter\Config\Factories::__callStatic('config', [])
 8 SYSTEMPATH\Common.php(208): CodeIgniter\Config\Factories::get('config', 'Config\\Filters')
 9 SYSTEMPATH\Config\Services.php(295): config('Config\\Filters')
10 SYSTEMPATH\Config\BaseService.php(312): CodeIgniter\Config\Services::filters(null, false)
11 SYSTEMPATH\Config\BaseService.php(251): CodeIgniter\Config\BaseService::__callStatic('filters', [...])
12 SYSTEMPATH\Config\Services.php(292): CodeIgniter\Config\BaseService::getSharedInstance('filters', null)
13 SYSTEMPATH\Config\BaseService.php(321): CodeIgniter\Config\Services::filters()
14 SYSTEMPATH\CodeIgniter.php(346): CodeIgniter\Config\BaseService::__callStatic('filters', [])
15 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
16 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
17 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-21 05:07:53 --> ParseError: syntax error, unexpected single-quoted string "SessionUser", expecting "]"
[Method: GET, Route: /]
in APPPATH\Config\Filters.php on line 14.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp82\\htdocs\\codeigniter4\\app\\Config\\Filters.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Filters')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Filters')
 4 SYSTEMPATH\Config\Factories.php(297): class_exists('Config\\Filters')
 5 SYSTEMPATH\Config\Factories.php(216): CodeIgniter\Config\Factories::locateClass([...], 'Config\\Filters')
 6 SYSTEMPATH\Config\Factories.php(159): CodeIgniter\Config\Factories::getDefinedInstance([...], 'Config\\Filters', [])
 7 SYSTEMPATH\Config\Factories.php(188): CodeIgniter\Config\Factories::__callStatic('config', [])
 8 SYSTEMPATH\Common.php(208): CodeIgniter\Config\Factories::get('config', 'Config\\Filters')
 9 SYSTEMPATH\Config\Services.php(295): config('Config\\Filters')
10 SYSTEMPATH\Config\BaseService.php(312): CodeIgniter\Config\Services::filters(null, false)
11 SYSTEMPATH\Config\BaseService.php(251): CodeIgniter\Config\BaseService::__callStatic('filters', [...])
12 SYSTEMPATH\Config\Services.php(292): CodeIgniter\Config\BaseService::getSharedInstance('filters', null)
13 SYSTEMPATH\Config\BaseService.php(321): CodeIgniter\Config\Services::filters()
14 SYSTEMPATH\CodeIgniter.php(346): CodeIgniter\Config\BaseService::__callStatic('filters', [])
15 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
16 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
17 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-21 05:07:53 --> ParseError: syntax error, unexpected single-quoted string "SessionUser", expecting "]"
[Method: GET, Route: /]
in APPPATH\Config\Filters.php on line 14.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp82\\htdocs\\codeigniter4\\app\\Config\\Filters.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Filters')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Filters')
 4 SYSTEMPATH\Config\Factories.php(297): class_exists('Config\\Filters')
 5 SYSTEMPATH\Config\Factories.php(216): CodeIgniter\Config\Factories::locateClass([...], 'Config\\Filters')
 6 SYSTEMPATH\Config\Factories.php(159): CodeIgniter\Config\Factories::getDefinedInstance([...], 'Config\\Filters', [])
 7 SYSTEMPATH\Config\Factories.php(188): CodeIgniter\Config\Factories::__callStatic('config', [])
 8 SYSTEMPATH\Common.php(208): CodeIgniter\Config\Factories::get('config', 'Config\\Filters')
 9 SYSTEMPATH\Config\Services.php(295): config('Config\\Filters')
10 SYSTEMPATH\Config\BaseService.php(312): CodeIgniter\Config\Services::filters(null, false)
11 SYSTEMPATH\Config\BaseService.php(251): CodeIgniter\Config\BaseService::__callStatic('filters', [...])
12 SYSTEMPATH\Config\Services.php(292): CodeIgniter\Config\BaseService::getSharedInstance('filters', null)
13 SYSTEMPATH\Config\BaseService.php(321): CodeIgniter\Config\Services::filters()
14 SYSTEMPATH\CodeIgniter.php(346): CodeIgniter\Config\BaseService::__callStatic('filters', [])
15 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
16 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
17 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-21 05:10:18 --> mysqli_sql_exception: Table 'login.t_usuarios' doesn't exist in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1836): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(2225): CodeIgniter\Database\BaseBuilder->batchExecute()
#5 C:\xampp82\htdocs\codeigniter4\app\Database\Seeds\Usuario.php(22): CodeIgniter\Database\BaseBuilder->insertBatch()
#6 C:\xampp82\htdocs\codeigniter4\system\Database\Seeder.php(147): App\Database\Seeds\Usuario->run()
#7 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Seed->run()
#9 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#12 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#13 {main}
ERROR - 2024-11-21 05:12:08 --> mysqli_sql_exception: Table 'login.t_usuarios' doesn't exist in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1836): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(2225): CodeIgniter\Database\BaseBuilder->batchExecute()
#5 C:\xampp82\htdocs\codeigniter4\app\Database\Seeds\Usuario.php(22): CodeIgniter\Database\BaseBuilder->insertBatch()
#6 C:\xampp82\htdocs\codeigniter4\system\Database\Seeder.php(147): App\Database\Seeds\Usuario->run()
#7 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Seed->run()
#9 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#12 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#13 {main}
CRITICAL - 2024-11-21 05:15:13 --> ErrorException: Namespace declaration statement has to be the very first statement or after any declare call in the script
[Method: CLI, Route: db:seed/Usuario]
in APPPATH\Database\Seeds\Usuario.php on line 1.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
ERROR - 2024-11-21 05:16:22 --> mysqli_sql_exception: Table 'login.t_usuarios' doesn't exist in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1836): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(2225): CodeIgniter\Database\BaseBuilder->batchExecute()
#5 C:\xampp82\htdocs\codeigniter4\app\Database\Seeds\Usuario.php(25): CodeIgniter\Database\BaseBuilder->insertBatch()
#6 C:\xampp82\htdocs\codeigniter4\system\Database\Seeder.php(147): App\Database\Seeds\Usuario->run()
#7 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Seed->run()
#9 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#12 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#13 {main}
ERROR - 2024-11-21 05:25:55 --> mysqli_sql_exception: Table 'login.t_usuarios' doesn't exist in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1836): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(2225): CodeIgniter\Database\BaseBuilder->batchExecute()
#5 C:\xampp82\htdocs\codeigniter4\app\Database\Seeds\Usuario.php(28): CodeIgniter\Database\BaseBuilder->insertBatch()
#6 C:\xampp82\htdocs\codeigniter4\system\Database\Seeder.php(147): App\Database\Seeds\Usuario->run()
#7 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Seed->run()
#9 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#12 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#13 {main}
ERROR - 2024-11-21 05:26:36 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`login`.`t_usuario_roles`, CONSTRAINT `t_usuario_roles_id_usuario_foreign` FOREIGN KEY (`id_usuario`) REFERENCES `t_usuario` (`id_usuario`) ON DELETE CASCADE ON UPDATE CASCADE) in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1836): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(2225): CodeIgniter\Database\BaseBuilder->batchExecute()
#5 C:\xampp82\htdocs\codeigniter4\app\Database\Seeds\TUsuarioRoles.php(16): CodeIgniter\Database\BaseBuilder->insertBatch()
#6 C:\xampp82\htdocs\codeigniter4\system\Database\Seeder.php(147): App\Database\Seeds\TUsuarioRoles->run()
#7 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Seed->run()
#9 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#12 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#13 {main}
ERROR - 2024-11-21 05:30:19 --> mysqli_sql_exception: Table 'login.t_usuarios' doesn't exist in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1836): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(2225): CodeIgniter\Database\BaseBuilder->batchExecute()
#5 C:\xampp82\htdocs\codeigniter4\app\Database\Seeds\Usuario.php(28): CodeIgniter\Database\BaseBuilder->insertBatch()
#6 C:\xampp82\htdocs\codeigniter4\system\Database\Seeder.php(147): App\Database\Seeds\Usuario->run()
#7 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Seed->run()
#9 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#12 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#13 {main}
ERROR - 2024-11-21 17:28:19 --> mysqli_sql_exception: Can't create table `login`.`t_usuario_roles` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\app\Database\Migrations\2024-11-21-041841_TUsuarioRoles.php(32): CodeIgniter\Database\Forge->createTable()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(867): App\Database\Migrations\TUsuarioRoles->up()
#6 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(183): CodeIgniter\Database\MigrationRunner->migrate()
#7 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run()
#9 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#12 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#13 {main}
ERROR - 2024-11-21 17:30:53 --> mysqli_sql_exception: Can't create table `login`.`t_usuario_roles` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\app\Database\Migrations\2024-11-21-041841_TUsuarioRoles.php(32): CodeIgniter\Database\Forge->createTable()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(867): App\Database\Migrations\TUsuarioRoles->up()
#6 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(183): CodeIgniter\Database\MigrationRunner->migrate()
#7 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run()
#9 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#12 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#13 {main}
ERROR - 2024-11-21 17:32:44 --> mysqli_sql_exception: Can't create table `login`.`t_usuario_roles` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\app\Database\Migrations\2024-11-21-041841_TUsuarioRoles.php(32): CodeIgniter\Database\Forge->createTable()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(867): App\Database\Migrations\TUsuarioRoles->up()
#6 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(183): CodeIgniter\Database\MigrationRunner->migrate()
#7 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run()
#9 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#12 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#13 {main}
ERROR - 2024-11-21 17:37:00 --> mysqli_sql_exception: Can't create table `login`.`t_usuario_roles` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\app\Database\Migrations\2024-11-21-041841_TUsuarioRoles.php(32): CodeIgniter\Database\Forge->createTable()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(867): App\Database\Migrations\TUsuarioRoles->up()
#6 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(183): CodeIgniter\Database\MigrationRunner->migrate()
#7 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run()
#9 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#12 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#13 {main}
ERROR - 2024-11-21 17:38:23 --> mysqli_sql_exception: Unknown table 'login.t_usuario_roles' in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\Forge.php(663): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\app\Database\Migrations\2024-11-21-041841_TUsuarioRoles.php(38): CodeIgniter\Database\Forge->dropTable()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(867): App\Database\Migrations\TUsuarioRoles->down()
#6 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(290): CodeIgniter\Database\MigrationRunner->migrate()
#7 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\MigrateRollback.php(101): CodeIgniter\Database\MigrationRunner->regress()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\MigrateRollback->run()
#9 C:\xampp82\htdocs\codeigniter4\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\MigrateRefresh.php(86): CodeIgniter\CLI\BaseCommand->call()
#11 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\MigrateRefresh->run()
#12 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#13 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#14 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#15 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#16 {main}
ERROR - 2024-11-21 17:38:23 --> mysqli_sql_exception: Can't create table `login`.`t_usuario_roles` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\app\Database\Migrations\2024-11-21-041841_TUsuarioRoles.php(32): CodeIgniter\Database\Forge->createTable()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(867): App\Database\Migrations\TUsuarioRoles->up()
#6 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(183): CodeIgniter\Database\MigrationRunner->migrate()
#7 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run()
#9 C:\xampp82\htdocs\codeigniter4\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\MigrateRefresh.php(87): CodeIgniter\CLI\BaseCommand->call()
#11 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\MigrateRefresh->run()
#12 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#13 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#14 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#15 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#16 {main}
ERROR - 2024-11-21 17:41:47 --> mysqli_sql_exception: Unknown table 'login.t_usuario_roles' in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\Forge.php(663): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\app\Database\Migrations\2024-11-21-041841_TUsuarioRoles.php(38): CodeIgniter\Database\Forge->dropTable()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(867): App\Database\Migrations\TUsuarioRoles->down()
#6 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(290): CodeIgniter\Database\MigrationRunner->migrate()
#7 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\MigrateRollback.php(101): CodeIgniter\Database\MigrationRunner->regress()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\MigrateRollback->run()
#9 C:\xampp82\htdocs\codeigniter4\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\MigrateRefresh.php(86): CodeIgniter\CLI\BaseCommand->call()
#11 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\MigrateRefresh->run()
#12 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#13 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#14 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#15 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#16 {main}
ERROR - 2024-11-21 17:41:48 --> mysqli_sql_exception: Can't create table `login`.`t_usuario_roles` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\app\Database\Migrations\2024-11-21-041841_TUsuarioRoles.php(32): CodeIgniter\Database\Forge->createTable()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(867): App\Database\Migrations\TUsuarioRoles->up()
#6 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(183): CodeIgniter\Database\MigrationRunner->migrate()
#7 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run()
#9 C:\xampp82\htdocs\codeigniter4\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\MigrateRefresh.php(87): CodeIgniter\CLI\BaseCommand->call()
#11 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\MigrateRefresh->run()
#12 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#13 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#14 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#15 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#16 {main}
ERROR - 2024-11-21 22:56:09 --> mysqli_sql_exception: Can't create table `login`.`t_usuario_roles` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\app\Database\Migrations\2024-11-21-041841_TUsuarioRoles.php(32): CodeIgniter\Database\Forge->createTable()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(867): App\Database\Migrations\TUsuarioRoles->up()
#6 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(183): CodeIgniter\Database\MigrationRunner->migrate()
#7 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run()
#9 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#12 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#13 {main}
ERROR - 2024-11-21 22:57:53 --> mysqli_sql_exception: Unknown table 'login.t_usuario_roles' in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\Forge.php(663): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\app\Database\Migrations\2024-11-21-041841_TUsuarioRoles.php(38): CodeIgniter\Database\Forge->dropTable()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(867): App\Database\Migrations\TUsuarioRoles->down()
#6 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(290): CodeIgniter\Database\MigrationRunner->migrate()
#7 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\MigrateRollback.php(101): CodeIgniter\Database\MigrationRunner->regress()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\MigrateRollback->run()
#9 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#12 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#13 {main}
ERROR - 2024-11-21 22:57:53 --> mysqli_sql_exception: Unknown table 'login.t_usuarios' in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\Forge.php(663): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\app\Database\Migrations\2020-12-04-202039_t_usuarios.php(35): CodeIgniter\Database\Forge->dropTable()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(867): App\Database\Migrations\TUsuarios->down()
#6 C:\xampp82\htdocs\codeigniter4\system\Database\MigrationRunner.php(290): CodeIgniter\Database\MigrationRunner->migrate()
#7 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\MigrateRollback.php(101): CodeIgniter\Database\MigrationRunner->regress()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\MigrateRollback->run()
#9 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#12 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#13 {main}
ERROR - 2024-11-21 22:59:29 --> mysqli_sql_exception: Table 'login.t_usuario' doesn't exist in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\app\Database\Seeds\Usuario.php(43): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\Seeder.php(147): App\Database\Seeds\Usuario->run()
#6 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call()
#7 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Seed->run()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#11 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#12 {main}
ERROR - 2024-11-21 22:59:29 --> mysqli_sql_exception: Table 'login.t_usuario' doesn't exist in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\app\Database\Seeds\Usuario.php(44): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\Seeder.php(147): App\Database\Seeds\Usuario->run()
#6 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call()
#7 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Seed->run()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#11 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#12 {main}
ERROR - 2024-11-21 22:59:29 --> mysqli_sql_exception: Table 'login.t_usuario' doesn't exist in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\app\Database\Seeds\Usuario.php(45): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp82\htdocs\codeigniter4\system\Database\Seeder.php(147): App\Database\Seeds\Usuario->run()
#6 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call()
#7 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Seed->run()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#11 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#12 {main}
ERROR - 2024-11-21 23:00:57 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`login`.`t_usuario_roles`, CONSTRAINT `t_usuario_roles_id_usuario_foreign` FOREIGN KEY (`id_usuario`) REFERENCES `t_usuarios` (`id_usuario`) ON DELETE CASCADE ON UPDATE CASCADE) in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query()
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute()
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1836): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(2225): CodeIgniter\Database\BaseBuilder->batchExecute()
#5 C:\xampp82\htdocs\codeigniter4\app\Database\Seeds\TUsuarioRoles.php(16): CodeIgniter\Database\BaseBuilder->insertBatch()
#6 C:\xampp82\htdocs\codeigniter4\system\Database\Seeder.php(147): App\Database\Seeds\TUsuarioRoles->run()
#7 C:\xampp82\htdocs\codeigniter4\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call()
#8 C:\xampp82\htdocs\codeigniter4\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Seed->run()
#9 C:\xampp82\htdocs\codeigniter4\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp82\htdocs\codeigniter4\system\Boot.php(104): CodeIgniter\Boot::runCommand()
#12 C:\xampp82\htdocs\codeigniter4\spark(84): CodeIgniter\Boot::bootSpark()
#13 {main}
ERROR - 2024-11-21 23:07:28 --> mysqli_sql_exception: Table 'login.t_usuario' doesn't exist in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp82\htdocs\codeigniter4\app\Models\Usuarios.php(8): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp82\htdocs\codeigniter4\app\Controllers\Home.php(21): App\Models\Usuarios->obtenerUsuario(Array)
#6 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Home->login()
#7 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp82\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
ERROR - 2024-11-21 23:07:28 --> La consulta a la base de datos falló en obtenerUsuario.
CRITICAL - 2024-11-21 23:07:28 --> TypeError: count(): Argument #1 ($value) must be of type Countable|array, bool given
[Method: POST, Route: login]
in APPPATH\Controllers\Home.php on line 23.
 1 APPPATH\Controllers\Home.php(23): count(false)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->login()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-21 23:07:38 --> mysqli_sql_exception: Table 'login.t_usuario' doesn't exist in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp82\htdocs\codeigniter4\app\Models\Usuarios.php(8): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp82\htdocs\codeigniter4\app\Controllers\Home.php(21): App\Models\Usuarios->obtenerUsuario(Array)
#6 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Home->login()
#7 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp82\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
ERROR - 2024-11-21 23:07:38 --> La consulta a la base de datos falló en obtenerUsuario.
CRITICAL - 2024-11-21 23:07:38 --> TypeError: count(): Argument #1 ($value) must be of type Countable|array, bool given
[Method: POST, Route: login]
in APPPATH\Controllers\Home.php on line 23.
 1 APPPATH\Controllers\Home.php(23): count(false)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->login()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-21 23:09:30 --> mysqli_sql_exception: Table 'login.t_usuario' doesn't exist in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp82\htdocs\codeigniter4\app\Models\Usuarios.php(8): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp82\htdocs\codeigniter4\app\Controllers\Home.php(21): App\Models\Usuarios->obtenerUsuario(Array)
#6 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Home->login()
#7 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp82\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
ERROR - 2024-11-21 23:09:30 --> La consulta a la base de datos falló en obtenerUsuario.
CRITICAL - 2024-11-21 23:09:30 --> TypeError: count(): Argument #1 ($value) must be of type Countable|array, bool given
[Method: POST, Route: login]
in APPPATH\Controllers\Home.php on line 23.
 1 APPPATH\Controllers\Home.php(23): count(false)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->login()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-21 23:09:34 --> mysqli_sql_exception: Table 'login.t_usuario' doesn't exist in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp82\htdocs\codeigniter4\app\Models\Usuarios.php(8): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp82\htdocs\codeigniter4\app\Controllers\Home.php(21): App\Models\Usuarios->obtenerUsuario(Array)
#6 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Home->login()
#7 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp82\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
ERROR - 2024-11-21 23:09:34 --> La consulta a la base de datos falló en obtenerUsuario.
CRITICAL - 2024-11-21 23:09:34 --> TypeError: count(): Argument #1 ($value) must be of type Countable|array, bool given
[Method: POST, Route: login]
in APPPATH\Controllers\Home.php on line 23.
 1 APPPATH\Controllers\Home.php(23): count(false)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->login()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-21 23:09:55 --> mysqli_sql_exception: Table 'login.t_usuario' doesn't exist in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp82\htdocs\codeigniter4\app\Models\Usuarios.php(8): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp82\htdocs\codeigniter4\app\Controllers\Home.php(21): App\Models\Usuarios->obtenerUsuario(Array)
#6 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Home->login()
#7 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp82\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
ERROR - 2024-11-21 23:09:55 --> La consulta a la base de datos falló en obtenerUsuario.
CRITICAL - 2024-11-21 23:09:55 --> TypeError: count(): Argument #1 ($value) must be of type Countable|array, bool given
[Method: POST, Route: login]
in APPPATH\Controllers\Home.php on line 23.
 1 APPPATH\Controllers\Home.php(23): count(false)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->login()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-21 23:17:57 --> mysqli_sql_exception: Table 'login.t_usuario' doesn't exist in C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp82\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(312): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp82\htdocs\codeigniter4\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp82\htdocs\codeigniter4\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp82\htdocs\codeigniter4\app\Models\Usuarios.php(8): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp82\htdocs\codeigniter4\app\Controllers\Home.php(21): App\Models\Usuarios->obtenerUsuario(Array)
#6 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Home->login()
#7 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp82\htdocs\codeigniter4\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp82\htdocs\codeigniter4\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp82\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp82\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
ERROR - 2024-11-21 23:17:57 --> La consulta a la base de datos falló en obtenerUsuario.
CRITICAL - 2024-11-21 23:17:57 --> TypeError: count(): Argument #1 ($value) must be of type Countable|array, bool given
[Method: POST, Route: login]
in APPPATH\Controllers\Home.php on line 23.
 1 APPPATH\Controllers\Home.php(23): count(false)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->login()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
